.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -4,4 +4,5 @@
 
 ## Contribution Guidelines
 
+- [ ] You have read the [Code Of Conduct](../../CODE_OF_CONDUCT.md) and promise to abide by these rules
```

Comment:
It should be written in first person ("I have..."), as the next point is written in the same way.

Reply:
Done.